design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -279,9 +190,8 @@ To hot-plug a new NIC into a running VMI, the user would execute the following
 command:
 ```bash
 $ virtctl addinterface <vmi-name> \
-    --name <kubevirt-spec-iface-name> \
-    --net <nad-name> \
-    --iface-name <desired-pod-interface-name> \
+    --network-name <nad-name> \
```

Comment:
A NAD is not a network, why not call it net-attach-def ?

Reply:
I think that's exposing someone else's detail into our API, hence I wanted to abstract it. 

Maybe `network` is so generic it doesn't mean anything, but I'd still for advocate something a bit more generic / conceptual.

Reply:
well, here I disagree. This is literally the [MultusNetwork](http://kubevirt.io/api-reference/main/definitions.html#_v1_multusnetwork) `networkName` attribute.

I'm just re-using the currently used name in the API.

Reply:
I thought we agreed it is a bad name, colliding with the VMI network and the network provider network (the one the NAD connects the interface to).

I would suggest to avoid riding on these bad names, it just increases the affected radius further.

Reply:
I think you're not considering the consistency of the API; it can be a bad name, but at least by now it is familiar. People will know what this relates to, because they're using it already somewhere else.

Hence, I think it is preferable to align all usages, then rename once we know the right thing to call them.